src/main/java/org/clas/viewer/EventViewer.java

```diff
@@ -874,6 +874,7 @@ public static void main(String[] args){
         parser.addOption("-trigger",  "0xFFFFFFFF",     "Select trigger bits");
         parser.addOption("-ethost",   "clondaq6",       "Select ET host name");
         parser.addOption("-etip",     "129.57.167.60",  "Select ET host name");
+        parser.addOption("-etsession","/et/clastest7",  "DAQ session, usually clasprod or clastest7");
```

Comment:
Let's leave the default value equal to clasprod, since it can be changed in the bin/mon12-rgl script as needed

Reply:
The GUI doesn't actually use this value at all... not sure why.